src/components/ChooseCard.js

```diff
@@ -0,0 +1,51 @@
+import React, {Component} from 'react';
+import './ChooseCard.css';
+
+class ChooseCard extends Component {
+    constructor(props) {
+        super(props)
+        this.state={
+            image : props.image,
+            name: props.name,
+            type: props.type,
+            description: props.desc,
+            atks:props.atks,
+            pv:props.pv
+        }
+    }
+    
+    render(){
+        return(
+            <section className="choose-card element-animation">
+                <div className="choose-card--content">
+                    <div className="choose-card--info">
+                        <span className="choose-card--info-name">{this.state.name}</span>
```

Comment:
Use destructuring to make your code clearer :

`const { name, hp, /* ... */ } = this.props;`